acquia/acquia_cms
# Drupal 10 readiness

**Motivation**
Make Acquia CMS Ecosystem Drupal 10 ready.

**Proposed changes**
Drupal 10 readiness

**Alternatives considered**
NA

**Testing steps**
- Use this branch
- Install site
- Install Upgrade status module
- Scan modules listed in ticket using upgrade status
- Verify upgrade status show it's compatible with Drupal 10
- Execute rector process
- Verify rector process show no issue
- Verify site working fine

**Merge requirements**
